clarity_ext/service/file_service.py

```diff
@@ -122,3 +127,40 @@ def cleanup(self):
 class SharedFileNotFound(Exception):
     pass
 
+
+class Csv:
+    """A simple wrapper for csv files"""
+    def __init__(self, file, delim=","):
```

Comment:
Yes, mine did too - I might have to turn that warning into an error to remember doing that. It's unfortunate here since this is actually expected to be a file-like object and this seems a good name for the instance. But, it is a built-in so I guess I'll just have to deal with that.